src/graph.jl

```diff
@@ -15,13 +15,92 @@ struct BaseGraph<:AbstractGraph
 end
 
 
+function polygon_array(coords::Vector{Vector{Vector{Vector{Float64}}}})::Array{LibGEOS.Polygon}
```

Comment:
Should polygon methods be in `graph.jl`? I would suggest moving them to a separate module (e.g. `geo.jl`).

Reply:
mmm yeah that makes sense to me! `graph` is getting a bit cluttered

Reply:
I'm going to make sure I resolve this critique once some of the upcoming PRs are merged 😅  otherwise it'll be kind of annoying to implement this, I think